site/content/en/docs/Examples/xonotic.md

```diff
@@ -0,0 +1,102 @@
+---
+
+## Prerequisite
+
+ To get started, ensure the following prerequisites are met:
+
+  - You have a running Kubernetes cluster.
+
+  - Agones is installed on your cluster. Refer to the [Agones guide](https://agones.dev/site/docs/installation/install-agones/) for the instructions.
+
+  - The Xonotic client downloaded for gameplay. Download it from [Xonotic](http://www.xonotic.org)
+
+  - (Optional) Review {{< ghlink href="examples/xonotic" >}}Xonotic code{{< /ghlink >}} to see the details of this example.
+
+
+## Create a Fleet
+
+Let's create a Fleet using the following command:
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/googleforgames/agones/{{< release-branch >}}/examples/xonotic/fleet.yaml
+```
+
+You should see a successful output similar to this :
+
+```
+fleet.agones.dev/xonotic created
+```
+
+This has created a Fleet record inside Kubernetes, which in turn creates two warm [GameServers]({{< ref "/docs/Reference/gameserver.md" >}})
+that are available to be allocated for a game session.
+
+```bash
+kubectl get fleet
+```
+It should look something like this:
+
+```
+NAME           SCHEDULING   DESIRED   CURRENT   ALLOCATED   READY   AGE
+xonotic        Packed       2         2         0           2       55s
+```
+
+You can also see the GameServers that have been created by the Fleet by running `kubectl get gameservers`,
+the GameServer will be prefixed by `xonotic`.
+
+```
+NAME                       STATE   ADDRESS          PORT   NODE                                        AGE
+xonotic-7lk8x-hgfrg        Ready   34.71.168.92     7206   gk3-genai-quickstart-pool-3-ba2a705f-wpmc   103s
+xonotic-7lk8x-rwhst        Ready   34.71.168.92     7330   gk3-genai-quickstart-pool-3-ba2a705f-wpmc   103s
+```
+
+For the full details of the YAML file head to the [Fleet Specification Guide]({{< ref "/docs/Reference/fleet.md" >}})
+
+{{< alert title="Note" color="info">}} The game servers deployed from a `Fleet` resource will be deployed in the same namespace. The above example omits specifying a namespace, which implies both the `Fleet` and the associated `GameServer` resources will be deployed to the `default` namespace. {{< /alert >}}
+
+## Viewing GameServer Logs
+
+For troubleshooting or to check how your game servers are running, you can look at the logs of a specific pod using:
+
+```bash
+kubectl logs -f <xonotic-game-server-pod-name>
+```
+
+## Connect to the Game Server
+
+After allocating a GameServer from the fleet and obtaining its status and IP, you're ready to connect and play. Here’s how to use the server IP and port to join the game with the Xonotic server:
+
+**Launch Xonotic**: Start the Xonotic client you previously downloaded by executing the `xonotic-linux64-sdl` or `xonotic-linux64-glx` script to launch and play the game.
```

Comment:
```suggestion
**Launch Xonotic**: Start the Xonotic client you previously downloaded by running the executable for your operating system ([documentation](https://xonotic.org/faq/#install)).
```

Since not everyone runs Linux 😄